source/cloud/azure/azureml.md

```diff
@@ -55,9 +65,147 @@ Launch the instance.
 
 ### Select the RAPIDS environment
 
-Once your Notebook Instance is `Running` select "JupyterLab"
+Once your Notebook Instance is `Running`, open "JupyterLab" and select the `rapids` kernel when working with a new notebook.
+
+## Azure ML Compute cluster
+
+Launch Azure's [ML Compute cluster](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-cluster?tabs=python) to distribute your RAPIDS training jobs across a cluster of single or multi-GPU compute nodes.
+
+The Compute cluster scales up automatically when a job is submitted, and executes in a containerized environment, packaging your model dependencies in a Docker container.
+
+### Instantiate workspace
+
+If using the Python SDK, connect to your workspace either by explicitly providing the workspace details or load from the `config.json` file downloaded in the pre-requisites section.
+
+```python
+from azure.ai.ml import MLClient
+from azure.identity import DefaultAzureCredential
+
+# Get a handle to the workspace
+ml_client = MLClient(
+    credential=DefaultAzureCredential(),
+    subscription_id="<SUBSCRIPTION_ID>",
+    resource_group_name="<RESOURCE_GROUP>",
+    workspace_name="<AML_WORKSPACE_NAME>",
+)
+
+# or load details from config file
+ml_client = MLClient.from_config(
+    credential=DefaultAzureCredential(),
+    path="config.json",
+)
+```
+
+### Create AMLCompute
+
+You will need to create a [compute target](https://learn.microsoft.com/en-us/azure/machine-learning/concept-compute-target?view=azureml-api-2#azure-machine-learning-compute-managed) using Azure ML managed compute ([AmlCompute](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-ai-ml/0.1.0b4/azure.ai.ml.entities.html)) for remote training. Note: Be sure to check limits within your available region. This [article](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-quotas?view=azureml-api-2#azure-machine-learning-compute) includes details on the default limits and how to request more quota.
+
+[**size**]: The VM family of the nodes.
+Specify from one of **NC_v2**, **NC_v3**, **ND** or **ND_v2** GPU virtual machines (e.g `Standard_NC12s_v3`)
+
+[**max_instances**]: The max number of nodes to autoscale up to when you run a job
+
+```{note}
+You may choose to use low-priority VMs to run your workloads. These VMs don't have guaranteed availability but allow you to take advantage of Azure's unused capacity at a significant cost savings. The amount of available capacity can vary based on size, region, time of day, and more.
+```
+
+```python
+from azure.ai.ml.entities import AmlCompute
+
+gpu_compute = AmlCompute(
+    name,
+    type="amlcompute",
+    size,
+    max_instances,
+    idle_time_before_scale_down,
+    tier="low_priority",  # optional
+)
```

Comment:
This code isn't valid. We should probably stub out the kwargs with some values fo folks to copy/modify.

```suggestion
gpu_compute = AmlCompute(
    name="rapids-example",
    type="amlcompute",
    size="Standard_NC12s_v3",
    max_instances=3,
    idle_time_before_scale_down=300,  # Seconds of idle time before scaling down
    tier="low_priority",  # optional
)
```